build.gradle

```diff
@@ -45,12 +45,6 @@ plugins {
 
 configurations.all {
   resolutionStrategy {
-    force("org.apache.httpcomponents.client5:httpclient5:5.4.4")
-    force("org.apache.httpcomponents:httpcore:5.3.4")
-    force("org.apache.httpcomponents.core5:httpcore5-h2:5.3.4")
-    force("commons-beanutils:commons-beanutils:1.11.0")
```

Comment:
How is this related to the CVE and why is this making it in the PR?

Reply:
This gets the version from core. Can you check similar change https://github.com/opensearch-project/opensearch-remote-metadata-sdk/pull/201